packages/kotti-ui/source/kotti-filters/components/FilterRow.vue

```diff
@@ -121,21 +124,35 @@ export default defineComponent<{
 	setup(props, { emit }) {
 		const translations = useTranslationNamespace('KtFilters')
 
-		const label = computed<string>(() =>
+		const isOperationSelectDisabled = computed(
+			() => operationOptions.value.length <= 1,
+		)
+		const isValueFieldVisible = computed(
+			() => !isEmptyOperation(props.filter.operation, props.column?.type),
+		)
+		const label = computed(() =>
 			props.isFirstItem
 				? translations.value.whereLabel
 				: translations.value.andLabel,
 		)
-		const operationOptions = computed<KottiFieldSingleSelect.Props['options']>(
-			() => {
-				if (!props.column) return []
-				return getOperationOptions(props.column)
-			},
-		)
+		const operationOptions = computed(() => {
+			if (!props.column) return []
+			return getOperationOptions(props.column)
+		})
 		const valueComponent = computed(() => {
 			if (!props.column?.type) return null
 			return getValueComponent(props.column.type)
 		})
+		const valueContainerClasses = computed(() => ({
+			'kt-filter__list__row__value-field': true,
+			'kt-filter__list__row__value-field--is-vertically-centered':
+				props.column?.type === KottiFilters.FilterType.BOOLEAN,
+		}))
+		const valueCurrency = computed(() =>
+			props.column?.type === KottiFilters.FilterType.CURRENCY
+				? props.column.currency
+				: null,
```

Comment:
Should be `: undefined` so that the prop isn’t passed at all to the components that don’t support it

Reply:
then it should be the case for `valuePrefix`, `valueSuffix`, `valueStep` as well